docs/mint.json

```diff
@@ -775,6 +775,7 @@
             "integrations/app-integrations/dockerhub",
             "integrations/app-integrations/email",
             "integrations/app-integrations/github",
+            "integrations/app-integrations/gitlab",
```

Comment:
Let's also add a line in this file: https://github.com/mindsdb/mindsdb/blob/staging/docs/integrations/data-sources-overview.mdx#L25
```
<Card title="GitLab" icon="link" href="/integrations/app-integrations/gitlab"></Card>
```